content/12.privacy.md

```diff
@@ -1,31 +1,32 @@
 ## Tip 10: Actively consider the ethical implications of your work {#privacy}
 
-While deep learning continues to be a powerful, transformative tool within life sciences research—spanning basic biology and pre-clinical science to varied translational approaches and clinical studies—it is important to comment on some ethics—related considerations.
-For instance, despite the fact that deep learning methods are helping to increase medical efficiency through improved diagnostic capability and risk assessment, certain biases may be inadvertently introduced into models related to patient age, race, and gender [@doi:10.1002/hast.977]; as previously mentioned, deep learning practitioners may make use of datasets not representative of diverse populations and patient characteristics [@doi:10.1377/hlthaff.2014.0048], thereby contributing to these problems (please refer to Tip 4).
+While deep learning continues to be a powerful, transformative tool within life sciences research—spanning basic biology and pre-clinical science to varied translational approaches and clinical studies—it is important to comment on ethical considerations.
+For instance, despite the fact that deep learning methods are helping to increase medical efficiency through improved diagnostic capability and risk assessment, certain biases may be inadvertently introduced into models related to patient age, race, and gender [@doi:10.1002/hast.977].
+Deep learning practitioners may make use of datasets not representative of diverse populations and patient characteristics [@doi:10.1377/hlthaff.2014.0048], thereby contributing to these problems.
 
-Therefore, it is important to think thoroughly and cautiously about deep learning applications and its potential impact to persons and society—mindful of possible harms, injuries, injustices, and other types of wrongdoings.
-At a minimum, practitioners must ensure that, wherever relevant, their life sciences projects are fully compliant with local research governance/approval policies, legal requirements, institutional review board (IRB) policies, and any other relevant bodies and their standards.
+Therefore, it is important to think thoroughly and cautiously about deep learning applications and their potential impact to persons and society—mindful of possible harms, injuries, injustices, and other types of wrongdoings.
+At a minimum, practitioners must ensure that, wherever relevant, their life sciences projects are fully compliant with local research governance and approval policies, legal requirements, institutional review board policies, and any other relevant bodies and standards.
```

Comment:
Please capitalize **Institutional Review Board** since it's a proper name.

Reply:
Are you sure it's a proper noun? Wikipedia lowercases it: https://en.wikipedia.org/wiki/Institutional_review_board

Reply:
In the field we always capitalize it as IRB... Perhaps it's always capitalized in practice since it's connected to a regulatory/compliance requirement. In this paper, we could go with either and see what reviewers say.

Reply:
See this section in the link you kindly posted: "Although 'IRB' is a generic term used in the United States by the FDA and HHS, each institution that establishes such a board may use whatever name it chooses. **Many simply capitalize the term 'Institutional Review Board' as the proper name of their instance.** Regardless of the name chosen, the IRB is subject to the FDA's IRB regulations when studies of FDA-regulated products are reviewed and approved."

Reply:
I capitalized it in c8d22f6.  I don't know the common convention but found several examples in a quick search where it is capitalized when used in a generic way.

Reply:
Fixed